internal/controllers/topology/cluster/desired_state.go

```diff
@@ -25,7 +25,6 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
Fine as a follow-up, but we should document how to do this in the ClusterClass docs.

Reply:
Which part of the docs do you think would this fit into?

I think for other fields we usually didn't write docs apart from godocs / descriptions for the CRD

Reply:
I'd do this in a follow-up PR :-)

Reply:
Yup let's just discuss first what we want to document where before we do it ;)